csv_detective/detect_labels/other/money/check_col_name.py

```diff
@@ -0,0 +1,11 @@
+def is_col_name_related_to_money (name):
+    #TODO : make this a little bit more clever (spacy ?)
+    col_name_related_to_money = False
+    money_themes = ['budget', 'salaire', 'euro', 'euros', 'prêt', 'montant'] # TODO attention 'européeen' est détecté OK
+    for theme in money_themes:
+        col_name_related_to_money = col_name_related_to_money or (theme in name)
```

Comment:
No problem for me but as we discussed, you will have some errors with this check. For instance, header "pays d'europe" would be associated with money as "euro" is contained in it. But could be improved later.